src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/ContaoWidgetManager.php

```diff
@@ -158,7 +159,7 @@ public function preLoadRichTextEditor()
                 continue;
             }
 
-            if (strncmp($extra['rte'], 'tiny', 4) !== 0) {
+            if (strncmp($extra['rte'], 'tiny', 4) !== 0 && strncmp($extra['rte'], 'ace', 3) !== 0) {
                 continue;
             }
 
```

Comment:
We should ensure here to only add each file once to solve the problem of adding the same config over and over again mentioned by @stefanheimes.